src/nodes/client/interactive.rs

```
@@ -172,13 +174,44 @@ impl ClientNode {
                 log::trace!("[OK] Got {} from GET", string);
                 writeln!(stdout, "{}", string)?;
             }
-            "PUT_SET" | "put_set" => {
+            "INC" | "inc" => {
+                let key = split
+                    .next()
+                    .ok_or_else(|| anyhow!("missing key and value arguments"))?;
+                let value = split
+                    .next()
+                    .ok_or_else(|| anyhow!("missing value argument"))?;
+                if let Some(extra) = split.next() {
+                    bail!("unexpected argument `{}`", extra);
+                }
+
+                let i = smol::block_on(self.inc(key.into(), value.parse()?))?;
+                writeln!(stdout, "{}", i)?;
+            }
+            "ADD_SET" | "add_set" => {
                 let key = split
                     .next()
                     .ok_or_else(|| anyhow!("missing key and value arguments"))?;
                 let value = split.map(|s| s.to_owned().into_bytes()).collect();
 
-                smol::block_on(self.put_set(key.into(), value))?;
+                smol::block_on(self.add_set(key.into(), value))?;
+                writeln!(stdout, "Success!")?;
+            }
+            "ADD_HASHMAP" | "add_hashmap" => {
+                let key = split
+                    .next()
+                    .ok_or_else(|| anyhow!("missing key and value arguments"))?;
+                let value = split
+                    .filter_map(|s| {
+                        if let Some((field, value)) = s.split_once(":") {
+                            Some((field.to_string(), value.to_string().into_bytes()))
+                        } else {
+                            None
```

This silently ignores values that use incorrect syntax. Perhaps it would be better to error instead?